job_runner/worker.py

```
@@ -0,0 +1,147 @@
+        {
+            "PROMPT": job["prompt"],
+            "REPO": job["repo"],
+            "BRANCH": job["branch"],
+            "TOKEN": job["token"],
+            "JOB_ID": job_id,
+            "CONVERSATION_HISTORY": json.dumps(job.get("conversation_history", [])),
+        }
+    )
+
+    logger.info(f"Starting local worker job {job_id}")
+    result = subprocess.run(
```

Ruff (0.15.10)

[warning] Async functions should not run processes with blocking methods

(ASYNC221)